lib/ruby_lsp/requests/support/common.rb

```diff
@@ -1,6 +1,9 @@
 # typed: strict
 # frozen_string_literal: true
 
+# WARNING: Methods in this class may be used by Ruby LSP extensions such as https://github.com/Shopify/ruby-lsp-rails,
+# or extensions by created by developers outside of Shopify, so be cautious of changing anything.
```

Comment:
Wouldn't this become the document of the top-level `RubyLsp`? Should it be directly above the `Common` class instead?

Reply:
Actually, this probably shouldn't appear anywhere in published docs, just the source, but I'm not sure if that's possible...